pod-operation/src/state_machine.rs

```diff
@@ -228,6 +231,10 @@ impl StateMachine {
 		{
 			return State::Faulted;
 		}
+		// Last 20% of the track, as indicated by braking
+		if self.lidar.read_distance() < END_OF_TRACK {
+			return State::Faulted;
```

Comment:
Question: should reaching the end of the track be conceptually considered a standard stop or a fault? I could understand faulting for if there's a sudden obstruction, but this seems equivalent to the standard stopping condition?

Reply:
maybe, but braking is looking to use this for testing purposes

Reply:
Noted, so in the future the threshold would be more like 1 meter.